client/src/test/java/io/pravega/client/stream/impl/EventStreamWriterTest.java

```diff
@@ -48,6 +49,13 @@
 import static org.mockito.ArgumentMatchers.eq;
 
 public class EventStreamWriterTest {
```

Comment:
Make this extend `ThreadPooledTestSuite`. It takes care of setting up and tearing down your executor without you having to write all that code. If you have it use a `0` thread pool it will instantiate an `InlineExecutor`.

Reply:
done